js/rpg_managers/ImageManager.js

```diff
@@ -9,6 +9,14 @@ function ImageManager() {
 
 ImageManager.cache = new CacheMap(ImageManager);
```

Comment:
I think it would be nice to add a comment that `ImageManager.cache` is no longer used but remained for plugins compatibility.